app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def welcome
+  end
+
+  def new
+      @user = User.new
+      # require 'pry';binding.pry
+  end
+
+  def create
+      user = user_params
+      user[:email] = user[:email].downcase
+      new_user = User.create(user)
+      flash[:success] = "Welcome, #{new_user.email}!"
+      redirect_to "/"
```

Comment:
We should decide if we're using path helpers or not for consistency!

Reply:
I think the project desc. said to use path helpers 'where possible'.  So we can do it both ways, but I agree consistency would be nicer.  There was no path name for the root path, but we can setup an alias I guess.